### PR TITLE
Add Collapse

### PR DESCRIPTION
**Pull request by martypowell**
Adds a component for a collapse (or singular accordion) for use with filtering inside of apps.

### CODE REVIEW

File: src/components/basic/Collpase.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (martypowell) [inline src/components/basic/Collpase.js]: This file needs to go.

Comment (tmccoy529) [reply src/components/basic/Collpase.js]: Hahaha spelling......again

Comment (martypowell) [reply src/components/basic/Collpase.js]: @tmccoy529 spelling is hard.

**Review by tmccoy529 — DISMISSED** [review]
Tested in the style guide as well. Looks good.

**Review by sgrg1 — APPROVED** [review]
looks good

**Pull request merged**